packages/x-date-pickers-pro/src/DateRangePicker/date-range-manager.ts

```diff
@@ -1,13 +1,15 @@
 import { MuiPickersAdapter } from '@mui/x-date-pickers/internals';
 import { DateRange } from '../internal/models';
 
-interface CalculateRangeChangeOptions<TDate> {
+interface CalculateRangePreviewOptions<TDate> {
```

Comment:
Is this change related to the current PR ?
Make sure it is compatible with #6763 since it touches the same parts of the code